Jamfile

```diff
@@ -556,10 +547,10 @@ lib gcc : : <name>gcc <link>static ;
 lib libiconv : : <name>iconv <link>shared <search>/usr/local/lib ;
 
 # openssl on linux/bsd/macos etc.
-lib gcrypt : : <name>gcrypt <link>shared <search>/opt/local/lib <search>/usr/local/lib : : <include>/opt/local/include <include>/usr/local/include ;
+lib gcrypt : : <name>gcrypt <link>shared <search>/usr/local/lib : : <include>/usr/local/include ;
 lib z : : <link>shared <name>z <search>/usr/lib ;
-lib crypto : : <name>crypto <link>shared <use>z <search>/opt/local/lib : : <include>/opt/local/include ;
-lib ssl : : <name>ssl <link>shared <use>crypto <search>/opt/local/lib : : <include>/opt/local/include ;
```

Comment:
for me, it actually hurts, because if you install a custom `openssl` in another route, and still have one installed there, it's problematic (it happened to me), I don't think this is a standard route anymore for `openssl`. This is not critical, it's just getting the build as warn-less as possible.